src/app/network/crypto.ts

```diff
@@ -117,6 +140,47 @@ export async function getEncryptedFile(
   ];
 }
 
+export function encryptAndUploadMultipartFileReadable(
```

Comment:
This name shoudn't include Upload in it. It should be:

`encryptAndChunkFileReadable`